### PR TITLE
CMSIS-NN: Update DW conv unit tests to new interface

### PR DESCRIPTION
**Pull request by felix-johnny**
Unit tests for DW conv are updated as per new interface
using structs. Two new test cases are added as well.

### CODE REVIEW

**Comment by felix-johnny** [comment]
@mansnils .. could you please review this PR?

**Comment by felix-johnny** [comment]
Depends on https://github.com/ARM-software/CMSIS_5/pull/930

File: CMSIS/NN/Include/arm_nn_types.h [hunk 1]
```diff
@@ -88,6 +88,19 @@ typedef struct
...
+typedef struct
+{
+    int32_t             input_offset;   /**< Zero value for the input tensor */
+    int32_t             output_offset;  /**< Zero value for the output tensor */
+    int32_t             ch_mult;        /**< Channel Multiplier. ch_mult * in_ch = out_ch */
+    cmsis_nn_tile       stride;
+    cmsis_nn_tile       padding;
+    cmsis_nn_tile       dilation;
+    cmsis_nn_activation activation;
+
```
Comment (gmiodice) [inline CMSIS/NN/Include/arm_nn_types.h:101]: I am afraid you need to remove this empty line to avoid merge conflicts

Comment (felix-johnny) [reply CMSIS/NN/Include/arm_nn_types.h:101]: I'll get https://github.com/ARM-software/CMSIS_5/pull/930 in first, rebase and then update this..

**Pull request merged**